utility/doc/E2E_FEATURE_LIST.md

```diff
@@ -0,0 +1,121 @@
+# E2E Features <!-- omit in toc -->
+
+- [Legend](#legend)
+- [A. E2E Relay](#a-e2e-relay)
+  - [1. Trustless Relay 🟡](#1-trustless-relay-)
+  - [2. Delegated Relay 🟡](#2-delegated-relay-)
+  - [3. Rate Limited Application 🟢🔴](#3-rate-limited-application-)
+- [B. E2E QoS](#b-e2e-qos)
+  - [1. QoS - Single Check 🟡](#1-qos---single-check-)
+  - [2. QoS - Test Score Submission 🟢🔴](#2-qos---test-score-submission-)
+  - [3. QoS - Report Card Aggregation 🟢🔴](#3-qos---report-card-aggregation-)
+- [C. E2E Rewards \& Distributions](#c-e2e-rewards--distributions)
+  - [1. Servicer Relay Rewards - Custodial 🟡](#1-servicer-relay-rewards---custodial-)
+  - [2. Servicer Relay Rewards - Non-Custodial 🟡](#2-servicer-relay-rewards---non-custodial-)
+  - [3. Servicer Relay Rewards - Rev-Share 🔵](#3-servicer-relay-rewards---rev-share-)
+  - [4. Validator - Block Rewards 🟡](#4-validator---block-rewards-)
+  - [5. Fisherman - QoS Submission Rewards 🟡](#5-fisherman---qos-submission-rewards-)
+- [D. E2E Burns \& Slashes](#d-e2e-burns--slashes)
+  - [1. Validator - Double-Sign Burn 🔴](#1-validator---double-sign-burn-)
+  - [2. Application - Application Quota Stake Burn 🟢🔴](#2-application---application-quota-stake-burn-)
+  - [3. Servicer - Inaccurate Challenged Response 🔵](#3-servicer---inaccurate-challenged-response-)
+  - [4. Fisherman - Missing Test Score Burn 🔴](#4-fisherman---missing-test-score-burn-)
+  - [5. Fisherman - Incomplete Test Score Burn 🔴](#5-fisherman---incomplete-test-score-burn-)
+- [E. E2E PoS Functionality](#e-e2e-pos-functionality)
+  - [1. Actor Staking 🟡](#1-actor-staking-)
+  - [2. Actor Unstaking 🟡](#2-actor-unstaking-)
+  - [3. Actor Unbonding 🟡](#3-actor-unbonding-)
+  - [4. Actor Jailing 🟡](#4-actor-jailing-)
+  - [5. Actor Unjailing 🟡](#5-actor-unjailing-)
+  - [6. Actor Pausing 🔵](#6-actor-pausing-)
+  - [7. Actor Unpausing 🔵](#7-actor-unpausing-)
+  - [8. Delegated Actor Staking 🔵](#8-delegated-actor-staking-)
+- [F. E2E Feature Flags](#f-e2e-feature-flags)
```

Comment:
cut from scope , can always introduce post launch

Reply:
Pushing back on this. We have this in v0 and what we need to safely upgrade the protocol and introduce new feature.

A must-have for MainNet launch